examples/asr/transcribe_speech.py

```diff
@@ -255,6 +257,14 @@ def main(cfg: TranscriptionConfig) -> Union[TranscriptionConfig, List[Hypothesis
     asr_model.set_trainer(trainer)
     asr_model = asr_model.eval()
 
+    if cfg.compute_dtype != "float32" and cfg.amp:
+        raise ValueError("amp=true is mutually exclusive with a compute_dtype other than float32")
```

Comment:
use nemo `logging.error`

Reply:
just to clarify, `raise ValueError` is commonly done throughout this file. Are you sure?

Reply:
Not sure of previous code, but usage of logging is the recommended way

Reply:
I'm not going to change to logging.error(). It's a critical error. Why shouls we continue execution in this case?